src/widgets/courses-school/ui/courses.module.scss

```diff
@@ -1,11 +1,6 @@
 .course-title {
   margin-top: 24px;
   margin-bottom: 0;
```

Comment:
Let's remove it too

Reply:
Please check courses.tsx 
        <WidgetTitle size="small" className={cx(' **_course-title_** ')}>Upcoming courses</WidgetTitle>